### DISCUSSION TITLE
Feature Request - Block Other screens

### DISCUSSION THREAD

**Discussion by Aboard89** [zidoro/pomatez#301] [Ideas]
Hi there,

I've just started using Pomatez and so far I think it's great. The only thing I've noticed is that if you are using multiple screens, the block screen only stays on the main screen. Is there a way of blocking all screens?

Many thanks,

Alex

**Comment by sekwah41** [comment]
I'm currently exploring switching the framework this is running on so a lot of the core things like that will alter how they are coded.

There is also already an issue for this raised though due to other work and issues have not had the time to implement this.
